prevalence_helper.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# TODO(blanchardjeremy): Need to check for the existence of the GitHub CLI before running
```

Comment:
if ! command -v gh &> /dev/null; then
    echo "GitHub CLI could not be found, please install from https://cli.github.com/"
    exit
fi

KEYSTORE=".secure-keys"
if [ ! -f "$KEYSTORE" ]; then
    echo "$KEYSTORE does not exist. Please obtain an API key from https://apidocs.covidactnow.org/"
    exit
fi

Reply:
@beshaya how do I then use $KEYSTORE? Do I still do `source .secure-keys` and access `$CAN_API_KEY` or do I access it through the keystore variable somehow?